app/src/androidTest/java/com/google/jetpackcamera/ImageCaptureDeviceTest.kt

```diff
@@ -51,71 +49,81 @@ internal class ImageCaptureDeviceTest {
     val permissionsRule: GrantPermissionRule =
         GrantPermissionRule.grant(*(APP_REQUIRED_PERMISSIONS).toTypedArray())
 
+    @get:Rule
+    val composeTestRule = createEmptyComposeRule()
+
     private val instrumentation = InstrumentationRegistry.getInstrumentation()
     private var activityScenario: ActivityScenario<MainActivity>? = null
```

Comment:
Does this need to be an instance variable or could it just be a local variable in each test?

Reply:
I just followed the convention in other test where this is an instance variable like this

Reply:
Sorry I'm seeing how Github is displaying my comment and it looks confusing. I'm talking about the `activityScenario` instance variable. It doesn't look like it's needed as an instance variable (or maybe at all).